test/test_helper.rb

```diff
@@ -180,6 +180,16 @@ def network_tokenization_credit_card(number = '4242424242424242', options = {})
       Billing::NetworkTokenizationCreditCard.new(defaults)
     end
 
+    # this mock is only useful with unit tests, as cryptograms generated by an EMV terminal
+    # are specific to the target acquirer, so remote tests using this mock will fail.
+    def credit_card_with_icc_data(options = {})
+      defaults = {
```

Comment:
Also, with this only being used for stripe unit tests right now - do you want to put it in the stripe test? Or is the hope that it's useful for other EMV implementation unit tests also?